test/aw.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "require": ["babel-register"],
+  "glob": ["test/unit/**/*.spec.js", "test/component/**/*.spec.js"],
```

Comment:
Is this glob needed when they're specified in `package.json`?

Reply:
Nope, the command line arg is overriding the one defined in the config. If not defined it will use `'test/**/*.spec.js'` as default

Reply:
@peol, @stoffeastrom This glob pattern is "default" and used by the `test` target. (`test:{unit | component | integration}` overrides it.)